src/operations.py

```diff
@@ -189,9 +189,9 @@ def shift_vertically(item, xdata, ydata, left_scale, right_scale, items):
         ymin = min(x for x in previous_ydata if x != 0)
         ymax = max(x for x in previous_ydata if x != 0)
         if item.yposition == "left":
-            linear = (left_scale == "linear")
+            linear = left_scale == "linear"
```

Comment:
Any specific reason for getting rid of the brackets? Personally I feel the brackets make it a bit more readable, as it distinguishes the comparison  (==) from the assignment (=) a bit visually.